### DISCUSSION TITLE
Expose battery in Kiosk Mode

### DISCUSSION THREAD

**Discussion by St3faan** [tijsverkoyen/HomeAssistant-FusionSolar#195] [Q&A]
I've set up the plugin in Kiosk mode. I see the solar panel energy and consumption, but I can't see my battery. In the documentation it says that I have to configure this, but I don't really see where exactly. Can someone help me?
many thanks!
S.

**Comment by tijsverkoyen** [comment]
The Battery data is not exposed.
I don't own a battery, so I can't tell if the data is exposed in later releases.
